piper/config.yaml

```diff
@@ -20,21 +20,17 @@ options:
   length_scale: 1.0
   noise_scale: 0.667
   noise_w: 0.333
-  max_piper_procs: 1
   debug_logging: false
   update_voices: true
-  streaming: true
```

Comment:
The Supervisor still has these removed options stored. To avoid warnings currently the recommended way is to delete the options explicitly in one of the add-on start scripts, see [the note in the options docs](https://developers.home-assistant.io/docs/add-ons/configuration#options--schema).

Reply:
Strange, definitly works here, tested with:
```
if bashio::config.exists 'autoflash_firmware'; then
    echo "Removing"                                                                                                        
    bashio::addon.option 'autoflash_firmware'
fi
```

Do you get an error? Or what happens?

It obviously only works once. Deleting every time leads a config save call to Supervisor every startup, which is certainly not ideal 😰

Reply:
Oh right, I see Supervisor filters the config according to schema, which essentially renders `bashio::config.exists` non working 🙈 

What does work is this:

```
options=$(bashio::addon.options)
key="test"
if bashio::jq.exists "${options}" ".${key}"; then
    bashio::log.info "Removing ${key}"
    bashio::addon.option '${key}'
fi
```

(note the dot before the key for the `jq.exists` call)

That said, I wonder if we should maybe make a breaking change in Supervisor and simply drop such orphan configs. The only downside would be that if an add-on accidentally removes a option, it can't simply readd it as system will have it removed 🤔

Reply:
I've updated it with the code you provided. I don't think it's a problem having to manually delete the old options, but we should at least fix the docs to use this new code.

Reply:
Right! Thanks for the hint.